### DISCUSSION TITLE
Directory for user settings, keybindings, etc. on Linux

### DISCUSSION THREAD

**Discussion by lcmen** [gitpod-io/openvscode-server#218] [Q&A] [answered]
Where does `openvscode-server` store its settings on Linux (when run natively via `./server.sh` script)?

I'd like to symlink or copy my existing VS Code settings to `openvscode-server` directory to share them.

**Comment by filiptronicek** [comment, marked as answer]
Does this answer your question? https://github.com/gitpod-io/openvscode-server/discussions/199#discussioncomment-1568487

**Reply by lcmen** [reply]
Yes, thanks!
